data/pa/people/David-Rowe-c88ccd9a-3378-4284-9dc6-767a4826526e.yml

```diff
@@ -0,0 +1,16 @@
+id: ocd-person/c88ccd9a-3378-4284-9dc6-767a4826526e
+name: "\u2018David\u2019 \u2018Rowe\u2019"
```

Comment:
if you remove the \u2018 \u2019 etc. stuff from all of the entries in this file it'll pass linting

Reply:
could you remove the extra \u2018/\u2019 chars from the name too?